packages/protocol/contracts/stability/SortedOracles.sol

```diff
@@ -1,29 +1,52 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
 pragma solidity ^0.5.13;
 
 import "openzeppelin-solidity/contracts/math/SafeMath.sol";
 import "openzeppelin-solidity/contracts/ownership/Ownable.sol";
 
 import "./interfaces/ISortedOracles.sol";
 import "../common/interfaces/ICeloVersionedContract.sol";
+import "./interfaces/IBreakerBox.sol";
 
 import "../common/FixidityLib.sol";
 import "../common/Initializable.sol";
 import "../common/linkedlists/AddressSortedLinkedListWithMedian.sol";
 import "../common/linkedlists/SortedLinkedListWithMedian.sol";
 
 /**
- * @title Maintains a sorted list of oracle exchange rates between CELO and other currencies.
+ * @title   SortedOracles
+ *
+ * @notice  This contract stores a collection of exchange rate reports between mento
+ *          collateral assets and other currencies. The most recent exchange rates
+ *          are gathered off-chain by oracles, who then use the `report` function to
+ *          submit the rates to this contract. Before submitting a rate report, an
+ *          oracle's address must be added to the `isOracle` mapping for a specific
+ *          rateFeedId, with the flag set to true. While submitting a report requires
+ *          an address to be added to the mapping, no additional permissions are needed
+ *          to read the reports, the calculated median rate, or the list of oracles.
+ *
+ * @dev     A unique rateFeedId identifies each exchange rate. In the initial implementation
+ *          of this contract, the rateFeedId was set as the address of the mento stable
+ *          asset contract that used the rate. However, this implementation has since
+ *          been updated, and the rateFeedId now refers to an address derived from the
+ *          concatenation of the mento collateral asset and the currency symbols' hash.
+ *          This change enables the contract to store multiple exchange rates for a
+ *          single stable asset. As a result of this change, there may be instances
+ *          where the term "token" is used in the contract code. These useages of the term
+ *          "token" are actually referring to the rateFeedId. You can refer to the Mento
+ *          protocol documentation to learn more about the rateFeedId and how it is derived.
```

Comment:
```suggestion
 * @notice  This contract stores a collection of exchange rates with of CELO
 *          expressed in units of other assets. The most recent exchange rates
 *          are gathered off-chain by oracles, who then use the `report` function to
 *          submit the rates to this contract. Before submitting a rate report, an
 *          oracle's address must be added to the `isOracle` mapping for a specific
 *          rateFeedId, with the flag set to true. While submitting a report requires
 *          an address to be added to the mapping, no additional permissions are needed
 *          to read the reports, the calculated median rate, or the list of oracles.
 *
 * @dev     A unique rateFeedId identifies each exchange rate. In the initial implementation
 *          of this contract, the rateFeedId was set as the address of the stable
 *          asset contract that used the rate. However, this implementation has since
 *          been updated, and the rateFeedId now also refers to an address derived from the
 *          concatenation other asset symbols. One project using this is Mento.
 *          This change enables the contract to store multiple exchange rates for a
 *          single token. As a result of this change, there may be instances
 *          where the term "token" is used in the contract code. These useages of the term
 *          "token" are actually referring to the rateFeedId. You can refer to the Mento
 *          protocol documentation to learn more about the rateFeedId and how it is derived.
```